specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -597,7 +597,7 @@
 			"x-ms-request-id": "request-id",
```

Comment:
The indentation above here is inconsistent with the rest of the file. Please make sure you're not mixing tabs and spaces (use whatever is used in the rest of the file).

Reply:
fixed